donate/templates/pages/core/campaign_page.html

```diff
@@ -1,3 +1,113 @@
 {% extends "pages/core/campaign_page_master.html" %}
+{% load static wagtailcore_tags wagtailimages_tags i18n %}
 
-{# There are no overrides in this template, but alternative apps can template-overide this file (e.g. thunderbird) #}
+{% block content %}
+
+<div class="layout layout__fundraise-up">
+    <div class="layout__container layout__container--full">
+        <div class="layout__full-col">
+            <div class="hero">
+                <div class="hero__container">
+                    <div class="hero__image">
+                        <picture>
+                        {% with alt_text=image.alt %}
+                            {% image page.hero_image width-1800 as imageDesktop %}
+                            {% image page.hero_image width-3600 as imageDesktop_2x %}
+                            <source media="(min-width: 1201px)" srcset="{{ imageDesktop.url }}, {{ imageDesktop_2x.url }} 2x"  alt="{{ alt_text }}">
+
+                            {% image page.hero_image width-1200 as imageTablet %}
+                            {% image page.hero_image width-2400 as imageTablet_2x %}
+                            <source media="(min-width: 768px)" srcset="{{ imageTablet.url }}, {{ imageTablet_2x.url }} 2x" alt="{{ alt_text }}">
+
+                            {% image page.hero_image width-768 as imageMobile %}
+                            {% image page.hero_image width-1536 as imageMobile_2x %}
+                            <source srcset="{{ imageMobile.url }}, {{ imageMobile_2x.url }} 2x" alt="{{ alt_text }}">
+
+                            <img src="{{ imageDesktop.url }}" class="hero__image-item" srcset="{{ imageMobile.url }} 768w, {{ imageTablet.url }} 1200w, {{ imageDesktop.url }} 1800w" alt="{{ alt_text }}"/>
+                        {% endwith %}
+                        </picture>
+                    </div>
+                    <div class="hero__content">
+                        <div class="hero__content-container">
+                            <h1 class="heading heading--primary hero__heading">{{ page.title }}</h1>
+                            <p class="hero__description">{{ page.lead_text }}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="sticky-message" data-waypoint-element>
+        <div class="sticky-message__container">
+            <a href="#anchor-donate" class="sticky-message__action button button--primary button--large">{% trans "DONATE NOW" context "Button" %}</a>
+        </div>
+    </div>
+    <div class="layout__container">
+        <div class="layout__half-col">
+            <div class="introduction column-spacing">
+                <h1 class="heading heading--primary">
+                    {% trans "Donate now" context "Header" %}
+                </h1>
```

Comment:
From Figma design it seems like we don't need to include this heading on campaign pages. Let's remove these lines.

Reply:
Hi @mmmavis! 

During a meeting that I had with Rebecca and Sabrina, they requested that I add the "Donate Now" header here above the introduction instead of above the donate form. I left a comment in the figma file, however the file itself does not reflect the request.

Tagging @beccaklam to see if she can please confirm 😄